linux_os/guide/system/accounts/accounts-session/accounts_users_home_files_groupownership/bash/shared.sh

```diff
@@ -0,0 +1,12 @@
+# platform = multi_platform_all
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
+
+for user in $(awk -F':' '{ if ($4 >= {{{ gid_min }}} && $4 != 65534) print $1 }' /etc/passwd); do
+    home_dir=$(getent passwd $user | cut -d: -f6)
+    group=$(getent passwd $user | cut -d: -f4)
+    # Only update the group-ownership when necessary.
+    find $home_dir -not -group $group -exec chgrp -f $group {} \;
```

Comment:
I wonder if this is necessary, wasn't the previous approach enough?
`awk -F':' '{ if ($4 >= {{{ gid_min }}} && $4 != 65534) system("chgrp -Rf " $4" "$6) }' /etc/passwd`

Reply:
Good point, thank you for the clarification and comment in the remediation.